lms/djangoapps/courseware/views/views.py

```diff
@@ -726,6 +726,12 @@ def _progress(request, course_key, student_id):
 
     # checking certificate generation configuration
     enrollment_mode, is_active = CourseEnrollment.enrollment_mode_for_user(student, course_key)
+
+    # If the learner is in verified modes and the student did not have
+    # their ID verified, we need to show message to ask learner to verify their ID first
+    must_be_id_verified = enrollment_mode in CourseMode.VERIFIED_MODES and \
```

Comment:
This is still confusing to me. If a user is enrolled in a verified mode, they must be ID-verified to receive their certificate; the validity of their verification has no bearing on the fact that they must be verified. If I'm understand this correctly, a more readable name might be `missing_required_verification`.